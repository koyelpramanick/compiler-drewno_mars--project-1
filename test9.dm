"\\""
BOOTY!
"you can escape a /b and leave the quote open!
silly too hot goose
"making sure 24Kmagic isn't recognized inside string"
fart != toot
return <=, >=, :
<; >; ? //wowwowowo
7--
