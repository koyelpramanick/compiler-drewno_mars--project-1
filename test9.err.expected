FATAL [1,5]-[1,7]: Unterminated string literal detected
FATAL [3,1]-[3,48]: Unterminated string literal detected
FATAL [8,7]-[8,8]: Illegal character ?
